debezium-server-iceberg-sink/src/main/java/io/debezium/server/iceberg/IcebergTableConfig.java

```diff
@@ -0,0 +1,5 @@
+package io.debezium.server.iceberg;
+
+public interface IcebergTableConfig {
+    String partitionBy();
```

Comment:
this could be like below, with this it will split comma separated values automatically.

Optional<List<String>> partitionBy();